elixir/apps/web/lib/web/live/gateways_live/index.ex

```diff
@@ -1,6 +1,25 @@
 defmodule Web.GatewaysLive.Index do
   use Web, :live_view
 
+  alias Domain.Gateways
+  alias Domain.Resources
+
+  def mount(_params, _session, socket) do
+    subject = socket.assigns.subject
+    {_, gateways} = Gateways.list_gateways(subject, preload: :group)
```

Comment:
We should never ignore the `:ok` or `:error` atoms in return tuples. Otherwise, `{:error, :anything}` will also match, `gateways` will be assigned to `:anything` and code will crash elsewhere:
```suggestion
    {:ok, gateways} = Gateways.list_gateways(subject, preload: :group)
```

Reply:
Gotcha.  I believe I did this to make sure there were no warnings being printed during the startup of the application.  Are we ok with having warnings like that during startup?

Reply:
I don't think there will be any warnings. In Elixir land it's very typical to match on `{:ok, whatever}` and "let it crash" otherwise.

Reply:
Cool.  I may have been misremembering about the warnings.  What you've mentioned makes sense. 👍

Reply:
This has been updated.